spec/controllers/lists_controller_spec.rb

```diff
@@ -0,0 +1,40 @@
+require 'rails_helper'
+
+
+describe ListsController do
```

Comment:
Currently the tests in this spec aren't using the `ListsController` action methods. Fix up the spec to use the controller action methods.